### DISCUSSION TITLE
Font rendering quality is bad when the layer isnt focused or zoomed.

### DISCUSSION THREAD

**Discussion by ilsubyeega** [uxbox/uxbox#3195] [Q&A] [answered]
I don't know how to replicate in other machines, but i'm concerned about this

https://user-images.githubusercontent.com/37479424/236630074-cb45ab8e-a105-4a81-92e7-13c91922b935.mp4

**Comment by superalex** [comment, marked as answer]
Thanks @ilsubyeega 

That's normal, as we have to render huge svg's the browser performance suffers a  lot. One of the things we do is using a thumbnail for the not active boards instead of rendering that part of the svg too.

**Reply by ilsubyeega** [reply]
uhh im a little surprised that it's normal for thumbnails being blurred. anyways thanks for answering!

**Reply by superalex** [reply]
It makes sense because thumbnails are not vectorial, when you zoom in or out in an image you have that blur/pixel effect. 

We have been working in this part a lot to make those thumbnails as perfect as we can but it has a limit.

Probably the most visible parts are the rendered fonts because of the aliasing.

You are welcome! :)
